docs/source/documentation/user-management.erb

```diff
@@ -0,0 +1,58 @@
+---
+title: User Management
+---
+
+## Adding users
+Use `./manage.py add-user` to add users
+
+```
+ Usage: manage.py add-user [OPTIONS] USERNAME
+
+Arguments
+*    username      TEXT  [default: None] [required]
+
+Options
+*  --email                        TEXT  [default: None] [required]
+*  --full-name                    TEXT  [default: None] [required]
+*  --password                     TEXT  [default: None] [required]
+  --disable      --no-disable          [default: no-disable]
+  --help                               Show this message and exit.
+
+
+Example: ./manage.py add-user test_user --email test.user@digital.justice.gov.uk --full-name "Test User" --password "password123"
+```
+
+##Updating users
```

Comment:
There needs to be a space after these hashtags for this heading to be rendered correctly.